lib/config/rules/typescript.js

```diff
@@ -101,7 +101,6 @@ module.exports = {
   'no-useless-constructor': 'off',
   'no-shadow': 'off',
   'no-use-before-define': 'off',
-  'sort-class-members/sort-class-members': 'off',
```

Comment:
Why is this being removed? 
For typescript, we enforce member ordering with [`typescript/member-ordering`](https://github.com/Shopify/eslint-plugin-shopify/blob/master/lib/config/rules/typescript.js#L48-L67), not eslint's equivalent.

Reply:
👍🏽 ah I totally missed that. I'll check it works and amend.

Reply:
👌🏽 fixed and rebased.